.github/workflows/pkgdown.yaml

```
@@ -0,0 +1,71 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  release:
+    types: [published]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+name: pkgdown
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    # Only restrict concurrency for non-PR jobs
+    concurrency:
+      group: pkgdown-${{ github.event_name != 'pull_request' || github.run_id }}
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: write
+      id-token: write
+      pages: write
+    steps:
+      - uses: actions/checkout@v6
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-pandoc@v2
+        with:
+          pandoc-version: "2.19.2"
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          cache: true
+          pak-version: rc
+          extra-packages: any::pkgdown, local::.
+          needs: website
+
+      - name: Build site
```

```suggestion
      - name: "Build site"
```

For other `name`s in this file, can you add "" to maintain consistency across other `.yaml` files in the hubs and `hubhelpr`. I believe this is an STF standard.

---

done https://github.com/CDCgov/hubhelpr/pull/78/commits/c1aad395ae16f03c1657c0544a327429af660f29